src/odemis/acq/align/test/goffset_test.py

```
@@ -32,6 +52,90 @@
+
+        # base gaussian peak
+        peak = 50 * np.exp(-0.5 * ((x - true_center) / 6.0) ** 2)
+
+        # simulate camera baseline
+        baseline = 300
+
+        # build 2D image with row variations
+        rows = 256
+        image = []
+
+        for i in range(rows):
```

Ruff (0.15.18)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)